static/src/js/App.jsx

```diff
@@ -72,12 +73,37 @@ const Subscriptions = lazy(() => import('./routes/Subscriptions/Subscriptions'))
 class App extends Component {
   constructor(props) {
     super(props)
-    this.state = {}
+    this.state = {
+      runTour: false
+    }
+
     this.store = configureStore()
     const { edscHost } = getEnvironmentConfig()
     const { env } = getApplicationConfig()
     this.edscHost = edscHost
     this.env = env
+    this.startTour = this.startTour.bind(this)
+    this.setRunTour = this.setRunTour.bind(this)
+  }
+
+  componentDidMount() {
+    const { disableSiteTour } = getApplicationConfig()
+    const isSiteTourEnabled = disableSiteTour === 'false'
+    const hasUserDisabledTour = localStorage.getItem('dontShowTour') === 'true'
+    const isLocalhost = window.location.hostname === 'localhost'
+    const shouldShowTour = isSiteTourEnabled && !hasUserDisabledTour && !isLocalhost
```

Comment:
Look into Playwright env variables to make the tour not run on each playwright test